pkg/render/certificatemanagement/certificatemanagement.go

```diff
@@ -98,7 +98,7 @@ func (c component) Objects() (objsToCreate, objsToDelete []client.Object) {
 			}
 			needsCSRRoleAndBinding = true
 		} else {
-			if keyPairCreator.renderInTruthNamespace && (!keyPair.BYO() || keyPair.GetName() == certificatemanagement.CASecretName) {
+			if keyPairCreator.renderInTruthNamespace && (!keyPair.BYO() || keyPair.GetName() != certificatemanagement.CASecretName) {
```

Comment:
Doesn't this break non-multitenancy?

Reply:
Yeah, this isn't the branch that is intended to handle multi-tenancy (although I can see why it would seem that way based on the name check).

Reply:
Talked offline. The CA generated per tenant does not have an issuer, but we still need to whitelist this in order to allow for multi-tenacy installation to happen. I have enhanced the condition.